state/state.go

```diff
@@ -1009,6 +1010,7 @@ func (s *State) Init() {
 	s.msgQueue = make(chan interfaces.IMsg, 50)                             //queue of Follower messages
 	s.prioritizedMsgQueue = make(chan interfaces.IMsg, 50)                  //a prioritized queue of Follower messages (from mmr.go)
 	s.MissingEntries = make(chan *MissingEntry, constants.INMSGQUEUE_HIGH)  //Entries I discover are missing from the database
+	s.dataQueue = NewInMsgQueue(constants.INMSGQUEUE_HIGH)                  //incoming requests for missing data
```

Comment:
@Emyrk ^ I probably have a bug here - this is a very large buffer

Reply:
ok not a bug exactly - maybe should be `make(chan interfaces.IMsg, constants.INMSGQUEUE_HIGH) ` instead